### PR TITLE
Various improvements

### PR DESCRIPTION
**Pull request by zoontek**
## What it does

- Add a valid SPDX license expression to `package.json` + fix the `LICENSE` file
- Format all files using `prettier`
- Add `lint-staged` to format all staged files
- Add `type input / output` to inputs and outputs to format and highlight TS correctly
- Switch to `jsonc` (https://github.com/github/linguist/issues/4153)
- Fix some typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,21 +1,19 @@
...
-
-
 
+<!-- [![CI Actions Status](https://github.com/Specifyapp/parsers/workflows/CI/badge.svg)](https://github.com/Specifyapp/parsers/actions) -->
 
 # Specify Parsers
 
 ## What is Specify?
-Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets—automatically. 
+
```
Comment (ChucKN0risK) [inline README.md:15]: Please remove line jump

File: README.md [hunk 2]
```diff
@@ -1,21 +1,19 @@
...
+Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets—automatically.
```
Comment (ChucKN0risK) [inline README.md:16]: Specify helps you unify your brand identity by collecting, storing and distributing design tokens and assets — automatically.

Specify is a design API facilitating the distribution of your design data within your design system.

design-api-full-dark

File: README.md [hunk 3]
```diff
@@ -38,6 +36,7 @@ Let's say you want to create a parser named `my-parser`.
 ### Creating a valid parser
 
 To be valid, your parser needs:
+
```
Comment (ChucKN0risK) [inline README.md:39]: Please remove line jump

File: README.md [hunk 4]
```diff
@@ -49,11 +48,13 @@ To be valid, your parser needs:
 ## Usable libraries
 
 For now, our parsers only use the following libraries:
+
```
Comment (ChucKN0risK) [inline README.md:51]: Please remove line jump

File: README.md [hunk 5]
```diff
@@ -49,11 +48,13 @@ To be valid, your parser needs:
...
 - [tinycolor2](https://github.com/bgrins/TinyColor)
 - [lodash](https://github.com/lodash/lodash)
 - [svgo](https://github.com/svg/svgo/)
 
 If you need another library to develop your parser:
+
```
Comment (ChucKN0risK) [inline README.md:57]: Please remove line jump

File: README.md [hunk 6]
```diff
@@ -62,7 +63,8 @@ If you need another library to develop your parser:
 To easily create and test your parsers, we advise you to use them on design tokens provided in the `seeds.json` file.
 It will allow you to use our fake tokens to test your parsers.
 
-To use our design tokens seed: 
+To use our design tokens seed:
+
```
Comment (ChucKN0risK) [inline README.md:67]: Please remove line jump

File: README.md [hunk 7]
```diff
@@ -49,11 +48,13 @@ To be valid, your parser needs:
 ## Usable libraries
 
 For now, our parsers only use the following libraries:
+
 - [tinycolor2](https://github.com/bgrins/TinyColor)
 - [lodash](https://github.com/lodash/lodash)
 - [svgo](https://github.com/svg/svgo/)
 
 If you need another library to develop your parser:
```
Comment (ChucKN0risK) [inline README.md:56]: How to add another library to develop your parser:

File: README.md (hunk 1 above)
Comment (zoontek) [reply README.md:15]: @ChucKN0risK You cannot with these prettier settings (it actually render the same as it's markdown)

File: .github/workflows/pull_request_template.md [hunk 8]
```diff
@@ -1,6 +1,5 @@
-
 ## What it does
 
-// Quick description
+<!-- Quick description -->
 
-## Additionnal informations
+## Additional informations
```
[the remark this answers is not in the document]
Comment (zoontek) [reply .github/workflows/pull_request_template.md:5]: @ChucKN0risK I guess there should be more than one information. It might be a list, no?

**Comment by zoontek** [comment]
@ChucKN0risK @AMoreaux I answered to remarks

**Comment by AMoreaux** [comment]
Thanks a lot for your contribution @zoontek 🎊 I just create a new PR #117  with your commits renamed to fit with our guidelines. Moreover, I omit changes on the license that require more discussion. Fill free to comment it 🙂

**Comment by zoontek** [comment]
@AMoreaux I just added a commit to fix code of conduct links (https://github.com/Specifyapp/parsers/pull/116/commits/7af44a6505effbc686831c3e50fd1dbe1a95cd10).

It's exactly the same licence, I just added a license property in `package.json` in a format that npm could understand (it must respect https://spdx.org/licenses) - because currently it says **"License: none"** on npm (https://www.npmjs.com/package/@specifyapp/parsers)

But also in a format that GitHub could understand (`LICENSE` file, not `LICENCE` which is the french word 😛) and not in markdown so it could display license details.

Example

Screenshot 2022-03-03 at 19 42 03

**Comment by AMoreaux** [comment]
Close in favor of #117

**Pull request closed without merging**